magefile.go

```diff
@@ -2176,6 +2184,9 @@ func createTestRunner(matrix bool, singleTest string, goTestFlags string, batche
 		extraEnv["AGENT_KEEP_INSTALLED"] = os.Getenv("AGENT_KEEP_INSTALLED")
 	}
 
+	extraEnv["TEST_EXTENDED"] = os.Getenv("TEST_EXTENDED")
+	extraEnv["LONG_TEST_RUNTIME"] = os.Getenv("LONG_TEST_RUNTIME")
```

Comment:
The mechanism we use to control this needs to be documented in https://github.com/elastic/elastic-agent/blob/main/docs/test-framework-dev-guide.md

`TEST_EXTENDED` is also pretty vague if you read it in isolation. Perhaps something like `TEST_LONG_RUNNING` and making it the inverse of `go test --short`. That said depending on how long this test actually takes it might not actually be that long if it is a few minutes, because the other tests aren't exactly fast.

Reply:
We could also just use a fixed naming convention for the long running tests and control it via the pattern passed to the `go test -run` argument.